docs/advanced_topics/settings.rst

```diff
@@ -317,6 +317,18 @@ Case-Insensitive Tags
 
 Tags are case-sensitive by default ('music' and 'Music' are treated as distinct tags). In many cases the reverse behaviour is preferable.
 
+Multi-word tags
+---------------
+
+.. code-block:: python
+
+  TAGGIT_SPACES_ALLOWED = False
```

Comment:
Should we rename this variable `TAG_SPACES_ALLOWED`? We don't need to expose the underlying tag implementation to site developers, and it's feasible that we'll switch to a different tagging library in the future.